beacon-chain/core/epoch/precompute/type.go

```diff
@@ -40,21 +40,21 @@ type Validator struct {
 // Balance stores the pre computation of the total participated balances for a given epoch
 // Pre computing and storing such record is essential for process epoch optimizations.
 type Balance struct {
-	// CurrentEpoch is the total effective balance of all active validators during current epoch.
-	CurrentEpoch uint64
-	// PrevEpoch is the total effective balance of all active validators during prev epoch.
-	PrevEpoch uint64
-	// CurrentEpochAttesters is the total effective balance of all validators who attested during current epoch.
-	CurrentEpochAttesters uint64
-	// CurrentEpochTargetAttesters is the total effective balance of all validators who attested
+	// CurrentEpochBal is the total effective balance of all active validators during current epoch.
```

Comment:
`Balance.CurrentEpochBal` sounds a bit odd...

How about `Balance.ActiveCurrentEpoch` ? The active/non active part was the source of the confusion anyway

Reply:
Some of the rename also came from it just being `p.CurrentEpoch`, maybe I'll try renaming that `p` to something else as well.

Reply:
I think the problem is `p`, we can change that to something better like `b`, `balance`... etc